src/cryptonote_config.h

```diff
@@ -50,12 +50,17 @@ using namespace std::literals;
 #define CRYPTONOTE_DEFAULT_TX_SPENDABLE_AGE             10
 #define CRYPTONOTE_DEFAULT_TX_MIXIN                     9
 
+#define STAKING_REQUIREMENT_HF16                        15000000000000
 #define STAKING_REQUIREMENT_LOCK_BLOCKS_EXCESS          20
-#define STAKING_PORTIONS                                UINT64_C(0xfffffffffffffffc)
-#define MAX_NUMBER_OF_CONTRIBUTORS                      4
-#define MIN_PORTIONS                                    (STAKING_PORTIONS / MAX_NUMBER_OF_CONTRIBUTORS)
-
-static_assert(STAKING_PORTIONS % 12 == 0, "Use a multiple of twelve, so that it divides evenly by two, three, or four contributors.");
+#define STAKING_PORTIONS_V1                             UINT64_C(0xfffffffffffffffc)
+#define STAKING_PORTIONS_V2                             STAKING_REQUIREMENT_HF16
```

Comment:
This needs to stay at the old value.  (Don't just revert to the previous 0xfffffffffffffff0 change in this PR).  If we change this base at all there is a good risk of introducing subtle changes into the calculations of rewards that could lead to consensus disagreements: in between the release and the hard fork we'll have some nodes calculating using 0xfff...c and some calculating using 0xfff...0 -- and if that introduces an off-by-one error we could have some subset of nodes that accept a registration and another that doesn't.

Reply:
(See my other comment about putting off the portions == requirement change for now; when we reintroduce that, we'll need a bunch of hardfork gating around which value gets used where).